spec/features/rooms/index_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'Rooms Index', type: :feature do
+  before(:each) do
+    @hotel = Hotel.create(name: 'Green Mountain Inn', location: 'Stowe')
+    
+    @room1 = @hotel.rooms.create(rate: 200, suite: 'Super')
+    @room2 = @hotel.rooms.create(rate: 100, suite: 'Deluxe')
+
+    @guest1 = Guest.create(name: 'Kye', nights: 6)
+    @guest2 = Guest.create(name: 'Martha', nights: 2)
+
+    GuestsRoom.create(guest: @guest1, room: @room1)
+    GuestsRoom.create(guest: @guest2, room: @room2)
+  end
+
+  it "displays a list of all rooms with suite, nightly rate, hotel name, and number of guests" do
+    visit "/rooms"
+
+    expect(page).to have_content(@room1.suite)
+    expect(page).to have_content(@room1.rate)
```

Comment:
These assertions would be more robust with `within` blocks and/or with assertions that include the fixed text around the dynamic values, i.e. `expect(page).to have_content("Nightly Rate: #{@room1.rate}")`
This helps us verify that the data is not only there, but also located in the right place rather than floating randomly on the page